data-labeling/environments/data-labeling/context/conda_dependencies.yaml

```
@@ -0,0 +1,17 @@
+name: data-labeling
+channels:
+- conda-forge
+- pytorch
+dependencies:
+- python=3.9
+- pip=21.3.1
+- pip:
+  - azureml-core=={{latest-pypi-version}}
+  - azureml-contrib-dataset=={{latest-pypi-version}}
+  - azureml-dataset-runtime[fuse,pandas]
+  - azureml-defaults=={{latest-pypi-version}}
+  - azure-storage-blob==12.14.1
+  - Pillow=={{latest-pypi-version}}
+  - pycairo==1.20.1
+  - pydicom=={{latest-pypi-version}}
+  - python-rle==0.0.3
```

Newline at end of file:
```suggestion
  - python-rle==0.0.3

```